src/components/content/common/ocl/FlavorsText.tsx

```diff
@@ -13,23 +13,30 @@ export function FlavorsText({ flavors }: { flavors: ServiceFlavor[] }): React.JS
     // These warnings must be suppressed because the Ocl object here is created from the import file and the data not necessarily contains all the mandatory fields.
     // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
     if (flavors) {
-        const flavorPops: React.JSX.Element[] = flavors.map((flavor, index) => {
+        const sortedFlavors = [...flavors].sort((a, b) => a.priority - b.priority);
+        const flavorPops: React.JSX.Element[] = sortedFlavors.map((flavor, index) => {
             const yamlDocument = new YAML.Document();
             // eslint-disable-next-line @typescript-eslint/ban-ts-comment
             // @ts-expect-error
             yamlDocument.contents = flavor;
             return (
-                <li key={index}>
+                <li key={index} className={oclDisplayStyles.oclFlavorItem}>
                     <Popover
                         content={
                             <pre className={oclDisplayStyles.oclFlavorsItemContent}>{yamlDocument.toString()}</pre>
                         }
                         title='Flavor Details'
                         trigger='hover'
                     >
-                        <Button className={oclDisplayStyles.oclDataHover} type='link'>
-                            {flavor.name}
-                        </Button>
+                        <Badge
```

Comment:
this is broken on small screens. The lines are extending outside the table and even the flavor names are outside the column space. 

If this not possible to fit flavors in the same table, you can consider adding flavor as a separate section below the `basic information`. and display flavors similar to new order flavor selection. (Except we do not need the prices inside it)

![image](https://github.com/eclipse-xpanse/xpanse-ui/assets/54129659/d24ade88-2f50-42b4-9521-b6f8b86e9afb)

Reply:
register:
![image](https://github.com/eclipse-xpanse/xpanse-ui/assets/42802200/c49c4269-4f2d-4d28-86f7-ad6314aa67ad)

catalog:
![image](https://github.com/eclipse-xpanse/xpanse-ui/assets/42802200/5246b56e-0e45-4280-88cb-03da62439f88)

review service details:
![image](https://github.com/eclipse-xpanse/xpanse-ui/assets/42802200/a5058869-344e-4e92-bfff-20fdc6b79120)

registered service:
![image](https://github.com/eclipse-xpanse/xpanse-ui/assets/42802200/9590f5d7-84b0-4b3e-8ed9-a0b2bd439332)